thali/thalireplicationmanager.js

```diff
@@ -112,6 +113,28 @@ ThaliReplicationManager.prototype.stop = function () {
   });
 };
 
+/**
+* Returns the current device's id if available. If not, the provided callback
+* function is saved and called when the id does become available.
+* @param {Function} cb the callback which returns the current device's id.
+*/
+ThaliReplicationManager.prototype.getDeviceIdentity = function (cb) {
+  if(currentDeviceIdentity) {
+    cb(null, currentDeviceIdentity);
+  } else {
+    var cryptomanager = require('./thalicryptomanager');
+    // get the device public-key-hash
```

Comment:
Please remove this comment. When the function is called 'getPublicKeyHash' it's pretty clear what's going on.